content/uiux/concepts/bias-in-user-research/bias-in-user-research.md

```diff
@@ -0,0 +1,111 @@
+
+When reviewing the data, the team noticed a clear trend: participants who talked to the younger researchers shared detailed responses that indicated honesty and openness. By contrast, most participants who spoke with the older researcher shared little detail, and their responses indicated an unwillingness to share openly. In this case, participants may have held a bias against the older researcher, which impacted how they approached the interview. </details> 
+
+### Response Bias 
+
+__Response bias__ is the tendency to consciously or subconsciously provide inaccurate information due to the nature of self-reporting. If the research study relies on self-reported data, it's important to keep in mind the limitations that come with it. For reasons that don't pertain to the design, the user response may be inaccurate or misleading. A user may misremember information, leave out details that would be important, or change their responses to portray a certain self-image.  
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+If a survey asks, “How much time do you typically spend on a popular social media app every day?”, participants may not know the exact amount. Their guess could be inaccurate, or they may report a lower number if they feel guilty about spending too much time on social media. To minimize response bias in this scenario, researchers need a way to obtain more accurate data. Some phones automatically track how much time is spent on each app. Asking participants to reference the data tracked by their phone should lead to more accurate responses. </details> 
+
+### Social Desirability Bias 
+
+__Social desirability bias__ is a tendency to provide answers that align with characteristics or viewpoints that the participant views as socially desirable. It is difficult to completely avoid these types of biases, as a participant will always know they are being interviewed, but researchers can minimize the effects by providing a comfortable environment, creating rapport and engaging in natural conversation throughout the interview, and keeping the participant's personal details anonymous. 
+
+<details open> 
+
+<summary>Example</summary> 
+
+<br> 
+
+Imagine that a researcher starts an interview with this question: “How often do you call your parents?” The participant may feel that they call their parents too frequently or infrequently. As a result, they might adjust their answer to match what they believe is normal or respectable.  
+
+To better lead into this topic, the interview could start with questions that are less personal, such as “In general, do you prefer texting or calling?” and “When you reach out to family members, how do you usually contact them?” Researchers should be mindful of topics that could spark social desirability bias. That way, they can carefully structure the study to help participants share openly and honestly. </details> 
```

Comment:
```suggestion
```html
<details open> 

<summary>Example</summary> 

<br> 

Imagine that a researcher starts an interview with this question: <q>How often do you call your parents?</q> The participant may feel that they call their parents too frequently or infrequently. As a result, they might adjust their answer to match what they believe is normal or respectable.  

To better lead into this topic, the interview could start with questions that are less personal, such as <q>In general, do you prefer texting or calling?</q> and <q> When you reach out to family members, how do you usually contact them?</q> Researchers should be mindful of topics that could spark social desirability bias. That way, they can carefully structure the study to help participants share openly and honestly. </details> 
```
```